voicebox/__main__.py

```diff
@@ -6,14 +6,21 @@
 
 
 logging.basicConfig(
-    level=logging.DEBUG,
+    level=logging.INFO,
     format='%(asctime)s %(name)s %(levelname)-8s  %(message)s',
     datefmt='(%H:%M:%S)'
 )
 
 
 def main():
 
+    """
+    Main function to start system
+
+    provides a command line interface for interracting
+    with the voicebox APIs.
+    """
+
     username = input("Username: ")
```

Comment:
In the `main` function, consider handling the case where the `input` function returns an empty string for the `username`. Currently, if the user doesn't enter a username and just presses enter, the program will continue with an empty username which might not be the intended behavior. [medium]